db/migrations/0002_alter_player_guild_alter_player_race_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-12-28 10:12
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='guild',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='db.guild'),
```

Comment:
The `guild` field in the Player model has been changed to a ForeignKey with `on_delete=DO_NOTHING`. While this allows multiple players to be associated with a single guild, consider using `SET_NULL` or `PROTECT` for better handling of deletions to avoid potential data integrity issues.